bin/graftM

```diff
@@ -129,60 +132,59 @@ if __name__ == '__main__':
+    graft is the main analysis pipeline of GraftM. To run graft, a GraftM package (gpkg) is required, the key components of which
+    are a HMM and DIAMOND database for searching  (with hmmsearch or DIAMOND) and a phylogenetic tree within which identified 
+    sequences are placed. Input sequence data may be provided in fasta (.fa) or fastq format, either gzipped (.fq.gz) or 
+    uncompressed. Protein or nucleotide sequences may be provided as input, and multiple datasets may be used in one run.    
     
- Example usage:
-     For a single file of reads:
-        $ graftM graft --forward my_reads.fa --graftm_package my_graftm_package.gpkg
-     
-     For paired reads:
-        $ graftM graft --forward my_forward_reads.fa --reverse my_reverse_reads.fa --graftm_package my_graftm_package.gpkg
-        
-     Using an assembly to create a "expand_search" database:
-        $ graftM graft --forward my_reads.fa --expand_search_contigs my_assembly_of_my_reads.fa --graftm_package my_graftm_package.gpkg
-
-
+                                                        ~ Example usage ~
+         For a single file of reads:
+            $ graftM graft --forward my_reads.fa --graftm_package my_graftm_package.gpkg
+         
+         For paired reads:
+            $ graftM graft --forward my_forward_reads.fa --reverse my_reverse_reads.fa --graftm_package my_graftm_package.gpkg
+            
+         Using an assembly to create a "expand_search" database:
+            $ graftM graft --forward my_reads.fa --expand_search_contigs my_assembly_of_my_reads.fa --graftm_package my_graftm_package.gpkg
         
 ''')
     input_options = graft_parser.add_argument_group('input options')
-    input_options.add_argument('--forward', nargs='+', metavar='forward_read', help='Forward read (or single read file), space separated in .fa, or .fq.gz format.', required=True)
-    input_options.add_argument('--reverse', nargs='+',metavar='reverse read', help='Reverse sequence file(s) in .fa, or .fq.gz format', default=None)
-    input_options.add_argument('--graftm_package', metavar='reference_package', help='Reference package of gene family')
+    input_options.add_argument('--forward', nargs='+', metavar='forward_read', help='Path to the reads you wish to run through GraftM, either in fasta (.fa) or fastq, gzipped format (.fq.gz). If you would like to run multiple samples at once, provide a space separated list of the file paths.', required=True)
+    input_options.add_argument('--reverse', nargs='+',metavar='reverse read', help='If you have paired end data, you may wish to provide the reverse reads. If you are running more than one dataset through, please ensure that the order of the files passed to the --forward and --reverse flags is preserved.', default=None)
+    input_options.add_argument('--graftm_package', metavar='reference_package', help='The path to the gene specific GraftM package containing the HMM, reference package etc.')
```

Comment:
It isn't consistent with "The path to .." vs "Path to .." from e.g. `--forward`. I prefer the latter.